README.ja.md

```diff
@@ -254,7 +254,9 @@ Server: WebSocket++/0.7.0
 
 この例を実行するには、`https://localhost:8443 `で WebRTC 互換ブラウザ を開き、ページをオープンするために、HTTPS トラフィックを受け入れます。アプリケーションは、2つの HTML5  `<video>` タグを含む、1つの HTML Web ページで構成されています。1つはローカル Web-cam でキャプチャされたローカル・ストリームと、もう1つはメディア・サーバからクライアントに送信されたリモート・ストリームです。"Start" ボタンをクリックすると、両方の `<video>` 要素に同じビデオが表示されます。
 
-![](https://fiware.github.io/tutorials.Media-Streams/img/hello-world-screenshot.png)
+[![](https://fiware.github.io/tutorials.Media-Streams/img/hello-world-screenshot.png)](https://www.youtube.com/watch?v=vGEnkSOp_xc "Hello World")
+
+Hello world の例のデモを見るには上の画像をクリックしてください。
```

Comment:
Could you add the video link for Hello world `https://www.youtube.com/watch?v=vGEnkSOp_xc` into the text?  I think it  disrupts the flow less when we have an ordinary link rather than an image to be clicked.